src/Patches/GhostHints.cs

```diff
@@ -61,6 +63,34 @@ public class HintGhost {
                 HintedItem = "";
                 OptionalCheckID = "";
             }
+            public HintGhost(string name, string sceneName, Vector3 position, Quaternion rotation, NPC.NPCAnimState animState, string dialogue, bool fishing, TransformData fishingRodPos) {
```

Comment:
Can we add these as optional parameters for the existing constructors? i.e.
`bool fishing = false, TransformData fishingRodPos = new TransformData()`

Reply:
Done